docker/k8s/README.md

```diff
@@ -27,14 +32,32 @@ in SGX-SIM and SGX-HW mode.
     kubectl create -f lmdb.yaml
     ```
 2. Start `EnclaveManager` (to manage Intel SGX enclaves)  
-   For Intel SGX simulator mode, use the below command
-    ```bash
-    kubectl create -f enclave-manager-deployment.yaml
-    ```
-   For Intel SGX hardware mode, use the below command
-    ```bash
-    kubectl create -f sgx-enclave-manager-deployment.yaml
-    ```
+   Singleton Mode (the same worker handles both keys and workloads):
+
+     For Intel SGX simulator mode, use the below command
+      ```bash
+      kubectl create -f enclave-manager-deployment.yaml
+      ```
+     For Intel SGX hardware mode, use the below command
+      ```bash
+      kubectl create -f sgx-enclave-manager-deployment.yaml
+      ```
+
+   Worker Pool Mode (with one Key Management Enclave and one Work order Processing Enclave):
+     Before running following commands, you need to get the MRENCLAVE value for WPE and replace
+     <WPE-MRENCLAVE-VALUE> with the MRENCLAVE value, you got from WPE, in enclave-manager-kme-deployment.yaml
```

Comment:
We could state the default MRENCLAVE value here for the time being whenever going into releases. This would ensure the defualt files are up and running and not surprise a first-time user. This would anyways be done away with, once we have a sequential binary building of WPE followed by KME. In that case, the MRENCLAVE would be handled implicitly and not require external intervention.

For the time being, we can add a FAQ for cases where a user tries to run a WPE with different MRENCLAVE. Steps involved would be -
1.) Modify the `docker-compose-pool.yaml` to update the `avalon-enclave-manager` service with appropriate values for `WORKLOADS` argument.
2.) Build the image using - `docker-compose -f docker-compose.yaml -f docker-compose-pool.yaml build avalon-enclave-manager`
3.) Start only the wpe to note down the MRENCLAVE. Only WPE could be started using - `docker-compose -f docker-compose.yaml -f docker-compose-pool.yaml up avalon-enclave-manager`
4.) Look for a log at the starting similar to -
`[04:58:36 INFO    avalon_enclave_manager.base_enclave_info] MRENCLAVE: 219AECFA1E621F269643373169DA5B314CB86547AC1984B4A5DCEE56140043DE`